phpstan.neon.dist

```diff
@@ -0,0 +1,10 @@
+parameters:
```

Comment:
To jsem ani nevedel, ze jde 👍

> If you do not provide a config file explicitly, PHPStan will look for files named phpstan.neon or phpstan.neon.dist in current directory.
>
> The resolution priority is as such:
>
> If config file is provided as a command line option, it will be used.
Otherwise, if phpstan.neon exists in the current working directory, it will be used.
Otherwise, if phpstan.neon.dist exists in the current working directory, it will be used.
If none of the above is true, no config will be used.